scripts/linters/test_files/invalid_test_only.py

```diff
@@ -0,0 +1,44 @@
+# coding: utf-8
```

Comment:
Why are these 2 files changed in this PR?

Reply:
I had earlier changed only the three files but some of the checks were failing due to stale build. I tried to fetch upstream and merge it into the branch I created. That led to the changes in these files. How do I correct this?